src/views/RegisterViews.vue

```diff
@@ -0,0 +1,279 @@
+            />
+          </div>
+        </div>
+        <div id="rightColumn">
+          <div class="inpt">
+            <label>Modelo</label>
+            <input
+              type="text"
+              placeholder="Ex: 777"
+              v-model="modelo"
+              required
+            />
+          </div>
+          <div class="inpt">
+            <label>Companhia</label>
+            <input
+              type="text"
+              placeholder="Ex: Latam"
+              v-model="companhia"
+              required
+            />
+          </div>
+          <div class="inpt">
+            <label>Numero de motores</label>
+            <input
+              type="text"
+              placeholder="Ex: 2"
+              v-model="numeroMotores"
+              required
+            />
+          </div>
+          <div class="inpt">
+            <label>Tipo de avião</label>
+            <input
+              type="text"
+              placeholder="Ex: Comercial"
+              v-model="tipoAviao"
+              required
+            />
+          </div>
+          <div class="inpt">
+            <label>País de origem</label>
+            <input
+              type="text"
+              placeholder="Ex: Brasil"
+              v-model="pais"
+              required
+            />
+          </div>
+        </div>
+      </div>
+      <div id="footer">
+        <button class="btn">
+          <router-link :to="{ name: 'list' }" class="linkVoltar"
+            >Voltar</router-link
+          >
+        </button>
+        <button class="btn">
+          <button class="btn enviar">
+            Enviar<i class="fa-solid fa-plane-departure icon"></i>
+          </button>
+        </button>
+      </div>
+    </form>
+    <div id="loadPage" v-show="load">
+      <LoadPage class="load" />
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+import LoadPage from "../components/LoadPage.vue";
+export default {
+  name: "RegisterViews",
+  components: {
+    LoadPage,
+  },
+  data() {
+    return {
+      load: false,
+      sucesso: false,
+
+      fabricante: "",
+      ano: "",
+      motor: "",
+      distancia: "",
+      prefixo: "",
+      modelo: "",
+      companhia: "",
+      numeroMotores: "",
+      tipoAviao: "",
+      pais: "",
+    };
+  },
+
+  methods: {
+    modal() {
+      this.load = true;
+      this.sucesso = true;
+      setTimeout(() => this.$router.push({ name: "list" }), 2000);
+    },
+
+    async createPlane() {
+      try {
+        const data = {
+          fabricante: this.fabricante,
+          ano: this.ano,
+          motor: this.motor,
+          distancia: this.distancia,
+          prefixo: this.prefixo,
+          modelo: this.modelo,
+          companhia: this.companhia,
+          numeroMotores: this.numeroMotores,
+          tipoAviao: this.tipoAviao,
+          pais: this.pais,
+        };
+        const res = axios.post("http://localhost:3000/airplanes", data);
```

Comment:
Não é necessario fazer um const data, caso tu tivesse deixado teus dados do data dentro de um object, tu simplesmente fazia : const res = "SERVICE".post("http://localhost:3000/airplanes", this.aviator);

Reply:
Também acho legal tu deixar teu loading nessa função assincrona para ser mais realista. e usar o finally para terminar o ciclo do loading: Exemplo 
   ```
   try {
        this.isLoading = true;
        const resp = await SERVICE.post();
      } catch (error) {
        alert(error);
      } finally {
        this.isLoading = false;
      }
```